sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/OpenTelemetryTracer.java

```diff
@@ -36,7 +36,26 @@
  */
 public class OpenTelemetryTracer implements com.azure.core.util.tracing.Tracer {
 
-    private final Tracer tracer = GlobalOpenTelemetry.getTracer("Azure-OpenTelemetry");
+    private final Tracer tracer;
+
+    /**
+     * Creates new {@code OpenTelemetryTracer} using default global tracer -
```

Comment:
Use {@link} instead of {@code} which will provide hyperlinks to navigate to the javadocs of the type/method.